src/TreeHouse/IoBundle/Item/Modifier/Data/Transformer/StringToDateTimeTransformer.php

```diff
@@ -2,45 +2,52 @@
 
 namespace TreeHouse\IoBundle\Item\Modifier\Data\Transformer;
 
-use TreeHouse\Feeder\Modifier\Data\Transformer\StringToDateTimeTransformer as FallbackTransformer;
+use DateTime;
+use TreeHouse\Feeder\Exception\TransformationFailedException;
 use TreeHouse\Feeder\Modifier\Data\Transformer\TransformerInterface;
 
 class StringToDateTimeTransformer implements TransformerInterface
 {
-    /**
-     * @var \DateTimeZone
-     */
-    protected $timezone;
+    /** @var \DateTimeZone */
+    private $timezone;
+    /** @var int */
+    private $minYear;
 
     /**
      * @param \DateTimeZone $timezone
      */
-    public function __construct(\DateTimeZone $timezone = null)
+    public function __construct(\DateTimeZone $timezone = null, int $minYear = 1900)
     {
         $this->timezone = $timezone ?: new \DateTimeZone('UTC');
+        $this->minYear = $minYear;
     }
 
     /**
      * @inheritdoc
      */
     public function transform($value)
     {
-        if (is_null($value) || empty($value)) {
+        if (empty($value)) {
             return null;
         }
 
-        if ($value instanceof \DateTime) {
+        if ($value instanceof DateTime) {
             return $value;
         }
 
-        // check if strtotime matches
-        if (false !== strtotime($value)) {
```

Comment:
This never worked for "invalid" dates like `0000-00-00` which for PHP is actually valid. Besides, the `DateTime` constructor already throws when the format is invalid.